rest-catalog/rest-catalog-service/src/main/java/org/projectnessie/restcatalog/service/resources/IcebergV1ApiResource.java

```diff
@@ -97,6 +98,7 @@ public class IcebergV1ApiResource extends BaseIcebergResource implements Iceberg
 
   @Override
   public ListNamespacesResponse listNamespaces(String prefix, String parent) throws IOException {
+    ThreadLocalBearerAuthentication.setAuthHeaderValue(authorization);
```

Comment:
Is there a way to avoid having these calls in every endpoint? If we miss one endpoint, it could use the authorization of another user.